modules/ask_va_api/config/locales/static_data.json

```diff
@@ -1,1974 +1,2350 @@
-          "RequiresAuthentication": false,
-          "AllowAttachments": false,
-          "RankOrder": 0,
-          "DisplayName": "Audiology and hearing aids",
-          "TopicType": "Topic",
-          "ContactPreferences": []
-      },
-      {
-          "Name": "General question",
-          "Id": "141a0e7f-8c34-ef11-8409-001dd809d228",
-          "ParentId": "62524deb-d864-eb11-bb24-000d3a579c45",
-          "Description": null,
-          "RequiresAuthentication": false,
-          "AllowAttachments": true,
-          "RankOrder": 0,
-          "DisplayName": "General question",
-          "TopicType": "Topic",
-          "ContactPreferences": []
-      },
-      {
-          "Name": "Programs and policies",
-          "Id": "bbaf4f8b-8c34-ef11-8409-001dd809d228",
-          "ParentId": "62524deb-d864-eb11-bb24-000d3a579c45",
-          "Description": null,
-          "RequiresAuthentication": false,
-          "AllowAttachments": true,
-          "RankOrder": 0,
-          "DisplayName": "Programs and policies",
-          "TopicType": "Topic",
-          "ContactPreferences": []
-      },
-      {
-          "Name": "Billing and copays",
-          "Id": "77bd7f33-773f-ef11-8409-001dd830980e",
-          "ParentId": "73524deb-d864-eb11-bb24-000d3a579c45",
-          "Description": null,
-          "RequiresAuthentication": false,
-          "AllowAttachments": true,
-          "RankOrder": 0,
-          "DisplayName": "Billing and copays",
-          "TopicType": "Topic",
-          "ContactPreferences": []
-      },
-      {
-          "Name": "Foreign Medical Program",
-          "Id": "b9d2b73f-773f-ef11-8409-001dd830980e",
-          "ParentId": "73524deb-d864-eb11-bb24-000d3a579c45",
-          "Description": null,
-          "RequiresAuthentication": false,
-          "AllowAttachments": true,
-          "RankOrder": 0,
-          "DisplayName": "Foreign Medical Program",
-          "TopicType": "Topic",
-          "ContactPreferences": []
-      },
-      {
-          "Name": "Eligibility for special benefits related to prosthetics",
-          "Id": "60b74965-773f-ef11-8409-001dd830980e",
-          "ParentId": "0d2b8586-e764-eb11-bb23-000d3a579c3f",
-          "Description": null,
-          "RequiresAuthentication": false,
-          "AllowAttachments": true,
-          "RankOrder": 0,
-          "DisplayName": "Eligibility for special benefits related to prosthetics",
-          "TopicType": "SubTopic",
-          "ContactPreferences": []
-      },
-      {
-          "Name": "Special benefits application status",
-          "Id": "0bfe416b-773f-ef11-8409-001dd830980e",
-          "ParentId": "0d2b8586-e764-eb11-bb23-000d3a579c3f",
-          "Description": null,
-          "RequiresAuthentication": false,
-          "AllowAttachments": true,
-          "RankOrder": 0,
-          "DisplayName": "Special benefits application status",
-          "TopicType": "SubTopic",
-          "ContactPreferences": []
-      }
+    {
```

Comment:
Can we just store this in the db instead? Does it have to be a jSON file? I see it's probably static, but does it ever change?

Reply:
This is a mock data file for development. My api doesn't use a db, I'm relying on CRM API as my db.